cypress/integration/inventory/item-status-date-updates.spec.js

```diff
@@ -0,0 +1,318 @@
+import NewLocations from '../../support/fragments/settings/tenant/locations/newLocation';
+import UsersEditPage from '../../support/fragments/users/usersEditPage';
+import Users from '../../support/fragments/users/users';
+
+describe('ui-inventory: Item status date updates', () => {
+  const instanceTitle = `autotestTitle ${Helper.getRandomBarcode()}`;
+  const itemQuantity = '1';
+  let orderNumber;
+  let effectiveLocationServicePoint;
+  let notEffectiveLocationServicePoint;
+  let effectiveLocation;
+  let user = {};
+  let userForDeliveryRequest = {};
+  let userBarcode = '';
+  const itemBarcode = Helper.getRandomBarcode();
+  let userRequestPreferences;
+
+  before(() => {,
+    // TODO rewrite with user diku_admin
+    cy.createTempUser([
+      permissions.uiCreateOrderAndOrderLine.gui,
+      permissions.uiEditOrderAndOrderLine.gui,
+      permissions.uiCanViewOrderAndOrderLine.gui,
+      permissions.inventoryAll.gui,
+      permissions.checkinAll.gui,
+      permissions.uiReceivingViewEditCreate.gui,
+      permissions.checkoutAll.gui,
+      permissions.requestsAll.gui,
+      permissions.loansAll.gui,
+      permissions.uiInventoryStorageModule.gui,
+      permissions.uiUsersDeclareItemLost.gui,
+      permissions.usersLoansRenewThroughOverride.gui,
+      permissions.uiUserEdit.gui
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        effectiveLocationServicePoint = NewServicePoint.getDefaulServicePoint();
+        notEffectiveLocationServicePoint = NewServicePoint.getDefaulServicePoint();
+
+        ServicePoints.createViaApi(effectiveLocationServicePoint);
+        ServicePoints.createViaApi(notEffectiveLocationServicePoint);
+        cy.addServicePointToUser([effectiveLocationServicePoint.id, notEffectiveLocationServicePoint.id],
+          user.userId, effectiveLocationServicePoint.id);
+
+        userRequestPreferences = {
+          id: uuid(),
+          fulfillment: 'Delivery',
+          defaultDeliveryAddressTypeId: null,
+          defaultServicePointId: effectiveLocationServicePoint.id,
+          delivery: true,
+          holdShelf: true,
+          userId: userForDeliveryRequest.id,
+        };
+
+        cy.login(userProperties.username, userProperties.password)
+          .then(() => {
+            NewLocations.createViaApi(NewLocations.getDefaultLocation(effectiveLocationServicePoint.id))
+              .then((location) => {
+                effectiveLocation = location;
+                Orders.createOrderWithOrderLineViaApi(
+                  NewOrder.getDefaultOrder(),
+                  BasicOrderLine.getDefaultOrderLine(itemQuantity, instanceTitle, location.id)
+                )
+                  .then(order => {
+                    orderNumber = order;
+                  });
+              });
+          });
+
+        cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
```

Comment:
as I see user was created before. It means you can catch all required properties and use it directly instead of extra get users requests. There are few such places. fix all of them

Reply:
Done